lib/archive.ts

```diff
@@ -95,7 +95,7 @@ async function copySourceToDest(
       if (!rootDir) {
         logger.debug(i18n(`${i18nKey}.copySourceToDest.sourceEmpty`));
         // Create the dest path if it doesn't already exist
-        fs.ensureDir(dest);
+        await fs.ensureDir(dest);
```

Comment:
This is an async method unless a callback is provided.  [Docs](https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir.md#ensuredirdiroptionscallback)